app/modelmodule/admin_acls.py

```diff
@@ -48,6 +49,10 @@ def get(repo, email_address):
         return AdminPermission.get_by_key_name(
             AdminPermission._key_name(repo, email_address))
 
+    @staticmethod
+    def get_for_repo(repo):
+        return AdminPermission.all().filter('repo =', repo)
```

Comment:
do we need .all() before .filter()?

Reply:
Weirdly yes, `all()` is one of the two ways to get a `Query` object to call `filter` on (the other way is less counter-intuitive, but Person Finder code is entirely consistent about its use of `all()` so I stick to it).